Sources/PointFree/TODO.swift

```diff
@@ -372,3 +372,40 @@ public func responseTimeout(_ interval: TimeInterval)
       }
     }
 }
+
+public struct Zipper<Element> {
```

Comment:
Simple `Zipper` type for now. Would be nice to build out to a library with more flexibility proper conformances and whatnot, but that's prob an episode!